llama-index-integrations/embeddings/llama-index-embeddings-bedrock/llama_index/embeddings/bedrock/base.py

```diff
@@ -38,7 +39,7 @@ class Models(str, Enum):
 
 
 class BedrockEmbedding(BaseEmbedding):
-    model: str = Field(description="The modelId of the Bedrock model to use.")
+    model_name: str = Field(description="The modelId of the Bedrock model to use.")
```

Comment:
we changed this, but still have `model=model` in the constructor code

Reply:
So should this not be changed? The point isn’t clear. 

It doesn’t make sense to have “model” used in some functions and “model_name” in others when they are both referencing the same value.

Reply:
Yea @nerdai is right, we changed to `model_name` here, but did not change the constructor

Reply:
Oh, I did miss those.  Rather than reverting to `model`, I went ahead and changed those to `model_name` as well.  The `BaseEmbedding`  parent class uses `model_name` ([here](https://github.com/run-llama/llama_index/blob/f9459c59767f5222527c36de4bca906e0df3dcfb/llama-index-core/llama_index/core/base/embeddings/base.py#L65)), and I don't see a justification for deviating from that in the Bedrock implementations.

Does that address the concerns?

Reply:
@logan-markewich Do the latest commits address this for you?  One of my projects is waiting on an evaluation between the two Titan embeddings.